tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "jsx": "react",
+        "target": "es6",
+        "module": "commonjs",
+        "rootDir": "src",
+        "outDir": "build",
+        "allowJs": true,
+        "pretty": true,
+        "moduleResolution": "node",
+        "strictNullChecks": false,
+        "experimentalDecorators": true
```

Comment:
Can you add https://github.com/danger/danger-js/blob/master/tsconfig.json#L6 - `"declaration": true,`

Then consumers get types 👍